src/app/components/AnswersWrapper.js

```diff
@@ -0,0 +1,25 @@
+import { Button } from './Button';
+
+export const AnswersWrapper = (answers, rightAnswer, chosenAnswer) => {
+  const answersWrapperObj = document.createElement('div');
+  answersWrapperObj.classList.add('answersWrapper');
+  answersWrapperObj.id = 'answersWrapper';
+
+  answers.forEach((answer, index) => {
+    const classList = ['button--answer'];
+    const isAnswerCorrect = answer === rightAnswer;
+    isAnswerCorrect
+      ? classList.push('button--correctAnswer')
+      : classList.push('button--incorrectAnswer');
+    const answerButton = Button({
+      id: `answer${index + 1}`,
+      btnText: answer,
+      classList,
+      onClickFn: chosenAnswer(answer, isAnswerCorrect),
+    });
+
+    answersWrapperObj.appendChild(answerButton);
+  });
```

Comment:
Nie można przekazywać oznaczać klas zanim użytkownik nie udzieli odpowiedzi.

Reply:
A to prawda - ktoś może sobie podejrzeć w kodzie, która jest poprawna :)

Reply:
Fakt, zastanowię się, jak to obejść.

Reply:
@lamparina Ja dodałem gotowy przycisk (jest w PR do akceptacji). 
Po nasłuchaniu odpowiedzi , że użytkownik odpowiedział na pytanie, powinna być uruchamiana funkcja w stylu "showCorrectnessBtns" w twoim wrapperze. Funkcja ta przydzielałaby odpowiednim przyciskom klase poprzez metodę w componencie Button.js. 
czyli
btnA.setCorrectAnswer();
btnA.setInorrectAnswer();
 
 Te metody sa juz stworzone i spowodują dodanie klasy (które sa już oscylowane - ale style mozesz zmienić na inne, jeśli inne Tobie lepiej pasują)

Reply:
Jak zaakceptujesz PR to pójdzie w świat i prześledzisz oraz docenisz sobie prostote  i kunszt tego rozwiąznia, oraz docenisz prostote użycia ;)